muted-tests.yml

```diff
@@ -118,9 +118,6 @@ tests:
 - class: org.elasticsearch.search.retriever.RankDocRetrieverBuilderIT
   method: testRankDocsRetrieverWithCollapse
   issue: https://github.com/elastic/elasticsearch/issues/112254
-- class: org.elasticsearch.datastreams.logsdb.qa.StandardVersusLogsIndexModeRandomDataChallengeRestIT
```

Comment:
I run into test failures when running: `./gradlew ":modules:data-streams:javaRestTest" --tests "org.elasticsearch.datastreams.logsdb.qa.StandardVersusLogsIndexModeRandomDataChallengeRestIT*" -Dtests.iters=32` from this branch.

For example: 

```
StandardVersusLogsIndexModeRandomDataChallengeRestIT > testTermsQuery {seed=[DE27E6D0B199E922:8E0DE99DE361A698]} FAILED
    org.elasticsearch.client.ResponseException: method [GET], host [http://[::1]:59848], URI [/logs-apache-contender/_search], status line [HTTP/1.1 500 Internal Server Error]
    {"error":{"root_cause":[{"type":"string_index_out_of_bounds_exception","reason":"Range [0, 11) out of bounds for length 9"}],"type":"search_phase_execution_exception","reason":"all shards failed","phase":"query","grouped":true,"failed_shards":[{"shard":0,"index":".ds-logs-apache-contender-2024.09.18-000001","node":"9fr1N7mtSwiH5tz9oUtyQw","reason":{"type":"string_index_out_of_bounds_exception","reason":"Range [0, 11) out of bounds for length 9","suppressed":[{"type":"illegal_state_exception","reason":"Failed to close the XContentBuilder","caused_by":{"type":"i_o_exception","reason":"Unclosed object or array found"}}]}}],"caused_by":{"type":"string_index_out_of_bounds_exception","reason":"Range [0, 11) out of bounds for length 9","caused_by":{"type":"string_index_out_of_bounds_exception","reason":"Range [0, 11) out of bounds for length 9","suppressed":[{"type":"illegal_state_exception","reason":"Failed to close the XContentBuilder","caused_by":{"type":"i_o_exception","reason":"Unclosed object or array found"}}]}}},"status":500}
        at __randomizedtesting.SeedInfo.seed([DE27E6D0B199E922:8E0DE99DE361A698]:0)
        at app//org.elasticsearch.client.RestClient.convertResponse(RestClient.java:351)
        at app//org.elasticsearch.client.RestClient.performRequest(RestClient.java:317)
        at app//org.elasticsearch.client.RestClient.performRequest(RestClient.java:292)
        at app//org.elasticsearch.datastreams.logsdb.qa.AbstractChallengeRestTest.query(AbstractChallengeRestTest.java:275)
        at app//org.elasticsearch.datastreams.logsdb.qa.AbstractChallengeRestTest.queryContender(AbstractChallengeRestTest.java:269)
        at app//org.elasticsearch.datastreams.logsdb.qa.StandardVersusLogsIndexModeChallengeRestIT.testTermsQuery(StandardVersusLogsIndexModeChallengeRestIT.java:215)
```

Reply:
Another instance: 

```
REPRODUCE WITH: ./gradlew ":modules:data-streams:javaRestTest" --tests "org.elasticsearch.datastreams.logsdb.qa.StandardVersusLogsIndexModeRandomDataChallengeRestIT.testTermsAggregation {seed=[FAB801CA9E14BC69:8FFB0D3C6FBF537D]}" -Dtests.seed=FAB801CA9E14BC69 -Dtests.locale=ne-Deva-NP -Dtests.timezone=Pacific/Fiji -Druntime.java=22

errors in baseline bulk response:
 {took=410, items=[{create={_index=.ds-standard-apache-baseline-2024.09.18-000001, _id=0, error={reason=[1:35350] mapping set to strict, dynamic introduction of [EKyrjlJ] within [_doc] is not allowed, type=strict_dynamic_mapping_exception}, status=400}}, {create={_index=.ds-standard-apache-baseline-2024.09.18-000001, _id=1, error={reason=[1:15448] mapping set to strict, dynamic introduction of [EKyrjlJ] within [_doc] is not allowed, type=strict_dynamic_mapping_exception}, status=400}}, {create={_index=.ds-standard-apache-baseline-2024.09.18-000001, _id=2, error={reason=[1:26355] mapping set to strict, dynamic introduction of [EKyrjlJ] within [_doc] is not allowed, type=strict_dynamic_mapping_exception}, status=400}}, {create={_index=.ds-standard-apache-baseline-2024.09.18-000001, _id=3, error={reason=[1:23832] mapping set to strict, dynamic introduction of [EKyrjlJ] within [_doc] is not allowed, type=strict_dynamic_mapping_exception}, status=400}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=0, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=4, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=1, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=5, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=2, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=6, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=3, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=7, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=4, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=8, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=5, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=9, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=6, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=10, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=7, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=11, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=8, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=12, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=9, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=13, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=10, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=14, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=11, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=15, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=12, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=16, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=13, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=17, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=14, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=18, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=15, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=19, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=16, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=20, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=17, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=21, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=18, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=22, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=19, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=23, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=20, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=24, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=21, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=25, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=22, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=26, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=23, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=27, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=24, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=28, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=25, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=29, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=26, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=30, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=27, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=31, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=28, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=32, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=29, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=33, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=30, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=34, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=31, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=35, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=32, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=36, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=33, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=37, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=34, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=38, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=35, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=39, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=36, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=40, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=37, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=41, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=38, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=42, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=39, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=43, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=40, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=44, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=41, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=45, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=42, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=46, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=43, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=47, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=44, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=48, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=45, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=49, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=46, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=50, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=47, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=51, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=48, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=52, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=49, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=53, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=50, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=54, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=51, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=55, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=52, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=56, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=53, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=57, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=54, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=58, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=55, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=59, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=56, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=60, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=57, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=61, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=58, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=62, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=59, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=63, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=60, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=64, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=61, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=65, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=62, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=66, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=63, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=67, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=64, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=68, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=65, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=69, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=66, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=70, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=67, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=71, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=68, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=72, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=69, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=73, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=70, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=74, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=71, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=75, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=72, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=76, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=73, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=77, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=74, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=78, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=75, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=79, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=76, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=80, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=77, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=81, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=78, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=82, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=79, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=83, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=80, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=84, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=81, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=85, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=82, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=86, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=83, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=87, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=84, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=88, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=85, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=89, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=86, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=90, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=87, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=91, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=88, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=92, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=89, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=93, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=90, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=94, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=91, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=95, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=92, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=96, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=93, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=97, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=94, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=98, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=95, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=99, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=96, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=100, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=97, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=101, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=98, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=102, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=99, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=103, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=100, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=104, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=101, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=105, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=102, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=106, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=103, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=107, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=104, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=108, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=105, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=109, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=106, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=110, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=107, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=111, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=108, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=112, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=109, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=113, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=110, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=114, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=111, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=115, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=112, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=116, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=113, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=117, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=114, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=118, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=115, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=119, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=116, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=120, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=117, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=121, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=118, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=122, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=119, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=123, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=120, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=124, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=121, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=125, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=122, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=126, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=123, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=127, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=124, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=128, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=125, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=129, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=126, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=130, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=127, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=131, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=128, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=132, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=129, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=133, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=130, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=134, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=131, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=135, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=132, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=136, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=133, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=137, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=134, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=138, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=135, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=139, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=136, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=140, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=137, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=141, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=138, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=142, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=139, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=143, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=140, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=144, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=141, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=145, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=142, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=146, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=143, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=147, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=144, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=148, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=145, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=149, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=146, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=150, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=147, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=151, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=148, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=152, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=149, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=153, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=150, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=154, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=151, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=155, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=152, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=156, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=153, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=157, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=154, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=158, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=155, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=159, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=156, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=160, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=157, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=161, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=158, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=162, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=159, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=163, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=160, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=164, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=161, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=165, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=162, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=166, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=163, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=167, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=164, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=168, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=165, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=169, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=166, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=170, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=167, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=171, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=168, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=172, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=169, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=173, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=170, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=174, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=171, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=175, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=172, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=176, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=173, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=177, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=174, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=178, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=175, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=179, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=176, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=180, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=177, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=181, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=178, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=182, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=179, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=183, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=180, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=184, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=181, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=185, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=182, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=186, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=183, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=187, _version=1, _primary_term=1, status=201}}], errors=true}
Expected: <false>
     but: was <true>
java.lang.AssertionError: errors in baseline bulk response:
 {took=410, items=[{create={_index=.ds-standard-apache-baseline-2024.09.18-000001, _id=0, error={reason=[1:35350] mapping set to strict, dynamic introduction of [EKyrjlJ] within [_doc] is not allowed, type=strict_dynamic_mapping_exception}, status=400}}, {create={_index=.ds-standard-apache-baseline-2024.09.18-000001, _id=1, error={reason=[1:15448] mapping set to strict, dynamic introduction of [EKyrjlJ] within [_doc] is not allowed, type=strict_dynamic_mapping_exception}, status=400}}, {create={_index=.ds-standard-apache-baseline-2024.09.18-000001, _id=2, error={reason=[1:26355] mapping set to strict, dynamic introduction of [EKyrjlJ] within [_doc] is not allowed, type=strict_dynamic_mapping_exception}, status=400}}, {create={_index=.ds-standard-apache-baseline-2024.09.18-000001, _id=3, error={reason=[1:23832] mapping set to strict, dynamic introduction of [EKyrjlJ] within [_doc] is not allowed, type=strict_dynamic_mapping_exception}, status=400}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=0, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=4, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=1, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=5, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=2, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=6, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=3, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=7, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=4, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=8, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=5, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=9, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=6, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=10, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=7, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=11, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=8, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=12, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=9, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=13, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=10, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=14, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=11, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=15, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=12, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=16, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=13, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=17, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=14, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=18, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=15, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=19, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=16, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=20, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=17, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=21, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=18, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=22, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=19, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=23, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=20, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=24, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=21, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=25, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=22, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=26, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=23, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=27, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=24, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=28, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=25, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=29, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=26, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=30, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=27, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=31, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=28, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=32, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=29, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=33, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=30, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=34, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=31, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=35, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=32, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=36, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=33, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=37, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=34, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=38, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=35, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=39, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=36, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=40, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=37, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=41, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=38, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=42, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=39, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=43, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=40, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=44, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=41, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=45, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=42, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=46, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=43, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=47, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=44, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=48, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=45, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=49, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=46, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=50, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=47, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=51, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=48, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=52, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=49, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=53, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=50, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=54, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=51, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=55, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=52, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=56, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=53, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=57, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=54, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=58, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=55, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=59, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=56, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=60, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=57, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=61, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=58, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=62, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=59, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=63, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=60, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=64, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=61, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=65, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=62, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=66, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=63, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=67, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=64, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=68, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=65, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=69, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=66, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=70, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=67, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=71, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=68, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=72, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=69, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=73, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=70, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=74, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=71, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=75, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=72, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=76, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=73, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=77, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=74, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=78, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=75, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=79, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=76, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=80, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=77, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=81, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=78, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=82, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=79, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=83, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=80, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=84, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=81, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=85, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=82, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=86, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=83, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=87, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=84, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=88, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=85, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=89, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=86, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=90, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=87, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=91, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=88, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=92, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=89, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=93, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=90, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=94, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=91, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=95, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=92, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=96, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=93, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=97, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=94, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=98, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=95, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=99, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=96, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=100, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=97, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=101, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=98, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=102, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=99, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=103, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=100, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=104, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=101, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=105, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=102, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=106, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=103, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=107, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=104, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=108, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=105, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=109, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=106, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=110, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=107, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=111, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=108, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=112, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=109, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=113, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=110, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=114, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=111, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=115, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=112, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=116, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=113, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=117, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=114, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=118, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=115, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=119, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=116, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=120, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=117, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=121, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=118, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=122, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=119, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=123, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=120, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=124, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=121, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=125, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=122, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=126, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=123, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=127, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=124, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=128, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=125, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=129, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=126, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=130, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=127, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=131, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=128, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=132, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=129, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=133, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=130, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=134, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=131, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=135, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=132, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=136, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=133, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=137, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=134, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=138, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=135, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=139, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=136, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=140, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=137, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=141, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=138, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=142, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=139, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=143, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=140, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=144, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=141, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=145, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=142, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=146, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=143, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=147, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=144, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=148, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=145, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=149, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=146, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=150, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=147, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=151, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=148, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=152, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=149, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=153, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=150, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=154, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=151, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=155, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=152, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=156, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=153, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=157, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=154, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=158, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=155, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=159, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=156, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=160, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=157, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=161, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=158, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=162, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=159, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=163, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=160, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=164, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=161, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=165, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=162, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=166, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=163, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=167, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=164, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=168, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=165, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=169, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=166, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=170, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=167, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=171, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=168, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=172, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=169, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=173, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=170, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=174, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=171, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=175, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=172, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=176, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=173, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=177, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=174, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=178, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=175, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=179, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=176, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=180, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=177, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=181, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=178, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=182, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=179, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=183, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=180, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=184, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=181, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=185, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=182, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=186, _version=1, _primary_term=1, status=201}}, {create={result=created, _shards={total=2, failed=0, successful=1}, _seq_no=183, _index=.ds-standard-apache-baseline-2024.09.18-000001, forced_refresh=true, _id=187, _version=1, _primary_term=1, status=201}}], errors=true}
Expected: <false>
     but: was <true>
	at __randomizedtesting.SeedInfo.seed([FAB801CA9E14BC69:8FFB0D3C6FBF537D]:0)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.elasticsearch.test.ESTestCase.assertThat(ESTestCase.java:2470)
	at org.elasticsearch.datastreams.logsdb.qa.StandardVersusLogsIndexModeChallengeRestIT.assertDocumentIndexing(StandardVersusLogsIndexModeChallengeRestIT.java:326)
	at org.elasticsearch.datastreams.logsdb.qa.StandardVersusLogsIndexModeChallengeRestIT.testTermsAggregation(StandardVersusLogsIndexModeChallengeRestIT.java:241)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
	at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
	at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
	at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
	at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
	at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
	at org.elasticsearch.test.cluster.local.DefaultLocalElasticsearchCluster$1.evaluate(DefaultLocalElasticsearchCluster.java:47)
	at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
	at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
	at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
	at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
	at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
	at java.base/java.lang.Thread.run(Thread.java:1570)
```

Reply:
Right, these are orthogonal issues, related to `copy_to`. Sasha is preparing a fix for the first one, and we need to update the data generator to avoid the second one, [known issue](https://github.com/elastic/elasticsearch/issues/113049).

Reply:
Ok - so then we have to wait before unmuting these tests until first issue is fixed and data generator is modified to avoid the second issue.

Reply:
I think we can unmute, they'll get muted again automatically as needed.

Reply:
Man I'm not awake.... was running tests from the wrong branch 🤦‍♂️ 

Looks like I'm no longer able to make these tests fail.

I think in general we shouldn't create unnecessary noise in normal CI and pr CI if we know failures are common. I think in this case, this isn't true.